components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -490,17 +488,15 @@ function RunningPrebuildView(props: RunningPrebuildViewProps) {
 
     return (
         <StartPage title="Prebuild in Progress">
-            <Suspense fallback={<div />}>
-                <WorkspaceLogs logsEmitter={logsEmitter} />
-            </Suspense>
-            <button
-                className="mt-6 secondary"
-                onClick={() => {
-                    props.onIgnorePrebuild();
-                }}
+            <PrebuildLogs
+                workspaceId={props.runningPrebuild.workspaceID}
+                onIgnorePrebuild={props.onIgnorePrebuild}
+                prebuildId={props.runningPrebuild.prebuildID}
             >
-                Don't Wait for Prebuild
-            </button>
+                <button className="secondary" onClick={() => props.onIgnorePrebuild && props.onIgnorePrebuild()}>
+                    Skip Prebuild
```

Comment:
Why change the text here?
Personally I don't have an opinion, but these texts tend to have been checked by other people having a better overview over UX then myself. :wink:

Reply:
Good point! I just followed the text specified in the issue because I assumed that went through a design discussion already. Thoughts, @jldec?

Reply:
:bulb: Ah, didn't noticed!